tests/__init__.py

```diff
@@ -163,3 +164,15 @@ def suspend_capture_stdin(pytestconfig: Any) -> Iterator[None]:
 
     if pytestconfig.getoption("capture") == "no":
         capmanager.resume_global_capture()
+
+
+@contextlib.contextmanager
+def ci_group(s: str) -> Iterator[None]:
+    try:
+        if os.getenv("GITHUB_ACTIONS"):
+            print("", flush=True)
+            print(f"::group::{s}", flush=True)
```

Comment:
This should be outside of the try

Reply:
I think that's visually a bit more noisy, but I don't have a strong preference.